src/hooks/useExplorerData.ts

```diff
@@ -18,52 +33,50 @@ export function useExplorerData() {
 
   const { nullifier, subsetRoot } = debouncedRecentWithdrawal;
   const { accessType, bitLength, subsetData: data } = subsetMetadata;
```

Comment:
In this chunk I wrapped the subset data into a useState/useEffect combo. I'm just not convinced Atom automagically knows to render if the network call is later than the react lifecycle.